aot/README.md

```diff
@@ -15,9 +15,14 @@ To enable use the AotMachine factory when building the module:
 // ...
 
 import com.dylibso.chicory.wasm.Parser;
-import com.dylibso.chicory.runtime.Instance;
-import com.dylibso.chicory.aot.AotMachine;
+import com.dylibso.chicory.experimental.aot.AotMachine;
 // ...
 var is = ClassLoader.getSystemClassLoader().getResourceAsStream("compiled/basic.c.wasm");
-Instance.builder(Parser.parse(is)).withMachineFactory(AotMachine::new).build();
+Instance.
+
+builder(Parser.parse(is)).
+
+withMachineFactory(AotMachine::new).
+
+build();
```

Comment:
```suggestion
Instance.builder(Parser.parse(is)).withMachineFactory(AotMachine::new).build();
```

Reply:
...but I can't see differences, maybe it's just something to revert

Reply:
fixed